src/xSTIR/pSTIR/STIR.py

```diff
@@ -2017,8 +2017,13 @@ def adjoint(self, ad, out=None):
            Added for CCPi CIL compatibility
            https://github.com/CCPPETMR/SIRF/pull/237#issuecomment-439894266
         '''
-        return self.backward(ad, subset_num=self.subset_num, 
-                             num_subsets=self.num_subsets, out=out)
+        if self.is_linear():
```

Comment:
@KrisThielemans am I right in thinking that this isn't neccessary? I will change the comment on the use of adjoint in the adjoint operator class in that case